### ISSUE TITLE
swagger response-types

### ISSUE THREAD

**Issue by marc0olo** [aeternity/aepp-middleware#204]
currently there are no response-types defined in the swagger.json which is included in the repository. it would be nice to have at least the type "object" as return-type.

best case would be to have schema definitions for each response-type.

**Comment by shekhar-shubhendu** [comment]
@marc0olo i agree and we need to do this. but currently, the API is not fully stable as we are focusing on providing a feature/functionality as soon as possible to the community.
Once we reach a stable state we will add schema definitions and try to complete the swagger.

**Comment by marc0olo** [comment]
do you plan to generate the schema definitions or would you define them by hand? just asking because you are asking for help

**Comment by shekhar-shubhendu** [comment]
@marc0olo not really sure at this point how we'll go about it. do you know any solution to automate this?

**Comment by marc0olo** [comment]
mhhh I just researched a little bit and there is some WIP project that addresses this:
- https://github.com/GREsau/okapi

the corresponding issue is:
- https://github.com/SergioBenitez/Rocket/issues/297

you are using rocket.rs right?
